pnet_base/src/macaddr.rs

```diff
@@ -13,15 +13,82 @@ use std::str::FromStr;
 #[cfg(feature = "serde")]
 use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 
+/// The number of bytes in an ethernet (MAC) address.
+pub const ETHER_ADDR_LEN: usize = 6;
+
+/// Structure of a 48-bit Ethernet address.
+pub type EtherAddr = [u8; ETHER_ADDR_LEN];
```

Comment:
I don't think this should be public - we don't expose anything else similar to this.

Reply:
no problem